src/mongo/connectToMongoClient.ts

```diff
@@ -12,7 +12,9 @@ export async function connectToMongoClient(connectionString: string, appName: st
         // appName should be wrapped in '@'s when trying to connect to a Mongo account, this doesn't effect the appendUserAgent string
         appName: `@${appName}@`,
         // https://github.com/lmammino/mongo-uri-builder/issues/2
-        useNewUrlParser: true
+        useNewUrlParser: true,
+        useUnifiedTopology: true,
+        serverSelectionTimeoutMS: 1000
```

Comment:
I'm gonna need more explanation for your changes

Reply:
The property `useUnifiedTopology` will be set to true by default from the next version mongodb. 
This property being true resolves the issue of "No primary found in replicaset or invalid replica set name." which appeared in the Nightly Build earlier (now resolved with recent updates) and is related to issue #1658 

It also resolves this warning... 
`DeprecationWarning: current Server Discovery and Monitoring engine is
deprecated, and will be removed in a future version. To use the new Server
Discover and Monitoring engine, pass option { useUnifiedTopology: true } to
the MongoClient constructor.
`

https://mongodb.github.io/node-mongodb-native/3.3/reference/unified-topology/
![image](https://user-images.githubusercontent.com/17537419/99600504-662bdc00-29b2-11eb-8fa2-4d578b477c4c.png)

![image](https://user-images.githubusercontent.com/17537419/99600569-8a87b880-29b2-11eb-95b3-b32abecbad41.png)

![image](https://user-images.githubusercontent.com/17537419/99600582-92475d00-29b2-11eb-813b-ce40420b58d1.png)

Reply:
Why not use the default `serverSelectionTimeoutMS`?

Reply:
@nturinski actually that would work too, I didn't notice the default time earlier. @ejizba What do you think?

Reply:
Yeah I would stick with the default unless you have some specific reason you want to change it